sync_check/config.toml

```diff
@@ -1,48 +0,0 @@
-# sync_check Configuration.
```

Comment:
hard to review the many files path changed, will it be compare with the new file path if use git mv **?